package.json

```diff
@@ -117,36 +118,59 @@
 			}
 		}
 	},
+	"npmPackageJsonLintConfig": {
+		"extends": "@wordpress/npm-package-json-lint-config",
+		"rules": {
+			"require-publishConfig": "error",
+			"valid-values-author": [
+				"error",
+				[
+					"WordPress"
```

Comment:
Nit: Out of curiosity, why did we decide upon `"WordPress"` as `author` when `wordpress-develop/package.json` specifies it as `"The WordPress Contributors"`.

https://github.com/WordPress/wordpress-develop/blob/ecb3302/package.json#L13

(cc @ntwb)

Reply:
Also: `Copyright 2011-2018 by the contributors` via

Reply:
Updated 👍